clash-lib/src/Clash/Primitives/DSL.hs

```diff
@@ -720,6 +730,24 @@ instHO bbCtx fPos (resTy, bbResTy) argsWithTypes = do
 
   pure (TExpr resTy (Identifier resName Nothing))
 
+compInBlock
```

Comment:
Document this function, maybe with an example on what it would look like in VHDL?